packages/aws-cdk-lib/aws-codepipeline/README.md

```diff
@@ -579,12 +579,17 @@ const rule = pipeline.notifyOnExecutionStateChange('NotifyOnExecutionStateChange
 
 ## Trigger
 
-To trigger a pipeline with Git tags, specify the `triggers` property. When a Git tag is pushed,
-your pipeline starts. You can filter with glob patterns. The `tagsExcludes` takes priority over
-the `tagsIncludes`.
-
+To trigger a pipeline with Git tags or branches, specify the `triggers` property.
 The triggers can only be used with pipeline type V2.
 
+### Push filter
```

Comment:
Not separated here by the chapters "Tags" and "Branches" to add pull request filters in [the PR](https://github.com/aws/aws-cdk/pull/29128).